tests/test_mad.py

```diff
@@ -0,0 +1,577 @@
+# -*- coding: utf-8 -*-
```

Comment:
This file doesn't contaitn tests. It should be renamed `mad.py` and placed into `piq.piq` folder.